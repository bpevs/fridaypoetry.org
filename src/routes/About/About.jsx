import { Col, Row } from "@zuck/core";
import * as React from "react";


export default function About(props) {
  return (
    <main className="info-page">
      <div className="origin-story">
        <h1 className="title">Friday Poetry</h1>
        <p>
          Friday poetry was started by Dan in the MakerSquare alumni Slack Channel
          (MakerSquare was a immersive programming course that no longer exists). Every week,
          people were encouraged to extend their creative power, and write some poetry! I
          enjoyed doing this because it stretched parts of my brain that I don't use on a
          regular basis. In fact, I enjoyed doing this so much, I decided to make it into a website.
        </p>
      </div>
      <Row>
        <Col>
          <img
            className="origin-image"
            alt="wordsmith"
            src="/assets/images/wordsmith.jpg" />
        </Col>
        <Col>
          <img
            className="origin-image"
            alt="the first professional poem"
            src="/assets/images/poem.jpg" />
        </Col>
      </Row>
      <div className="github-link">
        This website is
        {" "}
        <a href="https://github.com/ivebencrazy/fridaypoetry.org">open-source</a>
      </div>
    </main>
  );
}
