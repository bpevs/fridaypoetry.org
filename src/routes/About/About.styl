.info-page
  margin: auto
  padding: 2em

.origin-story
  color: black
  padding-left: 2em
  padding-right: 2em
  padding-bottom: 2em

  p
    padding-top: 0.5em

.origin-image
  width: 100%

.github-link
  padding-top: 2em
  padding-bottom: 2em
  text-align: center

  a
    text-decoration: none