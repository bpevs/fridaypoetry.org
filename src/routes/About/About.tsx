import { Component, h } from "preact";
require("./About");


interface Props {
  path?: string;
}


export default function About(props: Props) {
  return (
    <div class="info-page">
      <h1 class="title">About</h1>
      <div class="origin-story">
        <p>
          Friday poetry got its start by Dan in the MakerSquare alumni Slack Channel
          (MakerSquare was a coding immersive course that no longer exists). Every week,
          people were encouraged to extend their creative power, and write some poetry! I
          enjoyed doing this because it stretched parts of my brain that I don't use on a
          regular basis. In fact, I enjoyed doing this so much, I decided to make it into a website.
        </p>
        <p>This is that website.</p>
      </div>
    </div>
  );
}
