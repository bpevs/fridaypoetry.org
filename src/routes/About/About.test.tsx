import { h } from "preact";
import render from "preact-to-json";
import About from "./About";


test("Should render", () => {
  expect(render(<About />)).toMatchSnapshot();
});
