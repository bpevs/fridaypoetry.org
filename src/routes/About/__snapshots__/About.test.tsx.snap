// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 1`] = `
<main
  class="info-page"
>
  <div
    class="origin-story"
  >
    <h1
      class="title"
    >
      Friday Poetry
    </h1>
    <p>
      Friday poetry was started by Dan in the MakerSquare alumni Slack Channel (MakerSquare was a immersive programming course that no longer exists). Every week, people were encouraged to extend their creative power, and write some poetry! I enjoyed doing this because it stretched parts of my brain that I don't use on a regular basis. In fact, I enjoyed doing this so much, I decided to make it into a website.
    </p>
  </div>
  <Row>
    <Col>
      <img
        alt="wordsmith"
        class="origin-image"
        src="/assets/images/wordsmith.jpg"
      />
    </Col>
    <Col>
      <img
        alt="the first professional poem"
        class="origin-image"
        src="/assets/images/poem.jpg"
      />
    </Col>
  </Row>
  <div
    className="github-link"
  >
    This website is 
    <a
      href="https://github.com/ivebencrazy/fridaypoetry.org"
    >
      open-source
    </a>
  </div>
</main>
`;
